src/main/java/core/basesyntax/color/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.color;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+
+        return color.toString().toLowerCase();

```

Comment:
Please, do not use `toString()` to obtain a String representation of enum. Instead, use `name()` method.

[Code recommendations](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)